src/index.ts

```diff
@@ -1,21 +1,34 @@
-import {install as installLink, uninstall as uninstallLink} from './paste-markdown-image-link'
+import {install as installImageLink, uninstall as uninstallImageLink} from './paste-markdown-image-link'
+import {install as installLink, uninstall as uninstallLink} from './paste-markdown-link'
 import {install as installTable, uninstall as uninstallTable} from './paste-markdown-table'
 import {install as installText, uninstall as uninstallText} from './paste-markdown-text'
 
 interface Subscription {
   unsubscribe: () => void
 }
 
-export default function subscribe(el: HTMLElement): Subscription {
+function subscribe(el: HTMLElement): Subscription {
   installTable(el)
-  installLink(el)
+  installImageLink(el)
```

Comment:
What do you think about having all the `install*` functions in here so that consumers that use subscribe will get all the features, but then if they want to conditionally apply features they can import only the installers they plan on using?

Reply:
@koddsson I've thought about that as well. Can you confirm that [this line](https://github.com/github/github/pull/193520/files#diff-56fd9ba9b1627b3f8c4d982bc5106317f74d72fb0cdad47f025ac98d3df042efR12) is the only place where we consume the library at GitHub?

Since `subscribe` is now a named export, users of the library have to adjust the usage of the library either way, so yeah, I agree with adding the new feature to subscribe right away. We then only need to adjust [this line](https://github.com/github/github/pull/193520/files#diff-56fd9ba9b1627b3f8c4d982bc5106317f74d72fb0cdad47f025ac98d3df042efR12) as well to use the `add` and `remove` hooks as I do in [the lines below](https://github.com/github/github/pull/193520/files#diff-56fd9ba9b1627b3f8c4d982bc5106317f74d72fb0cdad47f025ac98d3df042efR14-R24) so that the new Link feature is not loaded by default.

Reply:
@koddsson I just saw [your proposal](https://github.com/github/github/pull/193520#discussion_r715459105) on using the `add` and `remove` hooks for all functions. I missed that earlier. I'll go with that approach. 👍 Thank you! 🙇‍♂️

Reply:
@koddsson I have applied your proposal here: https://github.com/github/paste-markdown/pull/23/commits/1c6b0cd957ececaa6d056c2dab554591dd3a3bca Thank you! 🙇‍♂️